types/proto/narwhal.proto

```diff
@@ -5,6 +5,124 @@ syntax = "proto3";
 
 package narwhal;
 
+// TODO: Make this proto the source of truth for all relevant narwhal types.
+
+/* 
+ * Proto wrapper for Narwhal type 
+ *
+ * pub struct CertificateDigest([u8; DIGEST_LEN])
+ *
+ */
+message CertificateDigest {
+    bytes f_bytes = 1;
```

Comment:
the name `f_bytes` isn't super descriptive I also don't know what `f_` means here.

Reply:
I just didn't want to reuse the message name as the bytes field name. So the `f_` was just to represent field and not use the field type `bytes` as the name. But I agree it's not clear, I changed the fields to just use a more descriptive name.